### DISCUSSION TITLE
Deny list challengers

### DISCUSSION THREAD

**Discussion by serbyxp** [helium/miner#1667] [Q&A]
First time this happened , I noticed I was challenged by someone on the deny list, hotspots on deny list are still able to challenge , and earn rewards for challenging, but I got 0 witnesses from it, is this a glitch ? If PoCv11 is limiting us to 3-4 beacons a day, shouldn’t we be guaranteed, to be challenged by valid hotspots?

*I also noticed the deny list is just a file in a folder, what happens if someone or a “group” of denied hotspots delete the file or modify it, will that “someone /or group” be able to bypass that deny list mechanism ? 
because clearly the validators , which I assume are the nodes meant to distribute HNt earnings, are still distributing the earnings to denied hotspots. 
There is a reason for those hotspots to be on the “deny list” it’s assumed that they are “gaming” the network, so it is assumed that they will “game” the mechanism implemented to “deny” them earnings. Even the ability to join / connect to the network.

I’m  beginning to believe , that these “centralized” not properly “proposed” for review / voted on changes to the network, aren’t properly thought out before implementation, causing more negative effects then positive ones, for the network.

*The deny list hotspots even earn beacon and witness rewards , they are far and in between, but challenging rewards are constant.

**Comment by serbyxp** [comment]
This issue still persists , I see deny listed hotspots earning more then the network average … not so much that It’s a big deal ( just thousands of hnt not available for equal distribution amongst the entire network, of legitimate participating hotspots) but it could be a much deeper rooted issue, since not denied hotspots are being challenged by deny listed hotspots, causing a reward imbalance. 

I haven’t tried it since I only got 1 hotspot , but I’m pretty sure they are just deleting the deny list file amongst themselves causing the filter not to run, or maybe just running an older version pre denylist with the updated v2 ledger… 

I think if the deny list had a hash attribute to it with some sort of checksum for the PoC , like how software programs have a hash check with the product key… that a mismatch or absence or manipulation of the file causes it to fail.

a venerability I have noticed but lack of personal knowledge on the subject, i noticed if you compile the parts ( modules nodes whatever ) of the miner software like. Libp2p portion separate then the blockchain handler. And /or use a central network drive to handle the blockchain data , and link all the miners to the one blockchain handler ( drive) it will work… so… you can have x amount of miners running off 1 blockchain db on a NAS say . And the ledger data also. Could potentially bypass the denylist . Or some “gamey” thing of the sort. 

The couple hotspots that have challanges me that are deny listed or , hotspots that I have seen in my logs that my miner filters out their witnesses, I have noticed they transfer the hotspots alot… not sure if that’s a glitch , but I mean it’s a ~$50 transfer fee after their 3rd attempt.. so how that balances out I dunno 🤷‍♀️
that or some one that buys it gets screwed and just keeps passing the hotspot down the line to another buyer that is unaware of this deny list upon purchasing the hotspot

**Comment by oosui** [comment]
Look at these hotspots, the People's Network will die soon... DeWi is a joke

https://explorer.helium.com/hotspots/11GGYuBoWQe2HEeH3q1pKF1C8xREx3WkduJ1utpgg2JNQDWzBa6
https://explorer.helium.com/hotspots/11nJ5wCuEJ7cZ3j8SqUjzegr8cnMYKURLxNDgigm3eXeceXofUo
https://explorer.helium.com/hotspots/11Qk1hQ9xMBE3wLFmuYj7fw5AfqNMRDWYi2ZQDG2oxY4PGu93jF
